### PR TITLE
fix(ticket): fix dropdown item ticket entity scope

### PR DESCRIPTION
**Pull request by stonebuzz**
Fix "My devices" dropdown.

image

Ex : 
All assets are in sub entity

When user create ticket from root entity and change it, page is reload on dropdown working fine (these assets are displayed).

But if you add item previously selected, dropdown is reload by ajax.

But loss entity scope, and asset list is empty

| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | !24434

### CODE REVIEW

**Review by orthagh — APPROVED** [review]

**Review by cedric-anne — APPROVED** [review]

File: src/Item_Ticket.php [hunk 1]
```diff
@@ -192,6 +192,7 @@ public static function itemAddForm(Ticket $ticket, $options = [])
 
         $params = [
             'id'  => (isset($ticket->fields['id']) && $ticket->fields['id'] != '') ? $ticket->fields['id'] : 0,
+            'entities_id'  => (isset($ticket->fields['entities_id']) && is_numeric($ticket->fields['entities_id']) ? $ticket->fields['entities_id'] : Session::getActiveEntity(),
```
Comment (trasher) [inline src/Item_Ticket.php:195]:
```suggestion
            'entities_id'  => (isset($ticket->fields['entities_id']) && is_numeric($ticket->fields['entities_id']) ? $ticket->fields['entities_id'] : Session::getActiveEntity()),
```
